clarifai/models/model_serving/cli/repository.py

```diff
@@ -40,17 +47,33 @@ def model_upload_init():
       required=True,
       help=f"Clarifai supported model types.\n Model-types-map: {MODEL_TYPES}",
   )
+  parser.add_argument(
+      "--image_shape",
+      type=dims_type,
+      default="[-1, -1]",
+      required=False,
+      help=
+      f"(H, W) dims for models with an image input type. H and W each have a max value of 1024",
+  )
   parser.add_argument(
       "--repo_dir",
       type=str,
-      default=".",  #curdir
+      default=".",
       required=True,
       help="Directory to create triton repository.")
 
   args = parser.parse_args()
+  if args.image_shape[0] or args.image_shape[1] > 1024:
```

Comment:
Wouldn't this always fail if someone provides a shape?
`args.image_shape[0]` would evaluate to True if it's not 0

Reply:
It evaluates to the same result as `if args.image_shape[0] > 1024 or args.image_shape[1] > 1024:`
going to update it though since it's not a proper boolean expression.